.prettierrc

```diff
@@ -5,5 +5,6 @@
   "semi": true,
   "tabWidth": 2,
   "trailingComma": "es5",
-  "bracketSpacing": true
+  "bracketSpacing": true,
+  "endOfLine": "crlf"
```

Comment:
i think we should stick with defaults of carriage return rather than windows style. i'd assume most people are using linux or macOs and only a handful are using windows. this may cause issue for linux and macos https://prettier.io/docs/en/options.html#end-of-line
![image](https://github.com/OpenBCca/openbc-web/assets/8355092/561856e8-86a6-442b-88a6-f604e5eed623)

Reply:
Just want to add, I'm using linux so looks like LF works better for me

Reply:
I will say, I don't care if we go one way or another.

The two things I want to prevent:
- Errors highlights when working locally
- Inconsistency that manifests as file changes in PRs

I have updated to us LF, I am slightly worried about others having to merge but we can support them as necessary. I have also updated our shared VS settings so that Unix style is default.

Reply:
i am not big fan of having IDE specific configurations in the repo for the following reason
1. we have prettier which sets up the formatting already of what the standard is
2. we have eslint to throw lint specific errors. 
having IDE specific configuration might help if everyone uses the same IDE.  it is highly unlikely everyone uses the same ide or prefers the same configuration. also the configuration might conflict with prettier formatting which in the end is a pain to fix. 
proposal, it is up for the dev to run prettier and lint however/whenever they want. but code that doesn't match these formats will be rejected. thus there will be a single tool to do one job and no possible way of having multiple tools fighting over each other trying to do the samething.

Reply:
I have the following opinion:
1) Dev should run the same local setup, if they choose to deviate then our pipelines are there to enforce the standards.
2) IDE settings are suppose to work in harmony with Prettier and EsLint. They are setup right now to use them as defaults for saving. The settings are automatically picked up when working in this workspace.
3) Everyone might have different preferences but these are team standards that should be followed. We can change standards based on team feedback but we must have single agreed upon standards.

I hear you about ensuring everything is configured so they all agree with each other. However, the consistency and code quality is worth it IMO.